managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,433 @@
+3. Click **Save**.
+4. Restart the application.
+
+When the application is restarted, the *Embedded Workflows* functionality is enabled. <!-- This functionality reuses existing API. -->
+
+### Embedded Workflows UI additions
+
+When the *Embedded Workflows* functionality is enabled, new **Automation** > **Embedded Workflows** pages become available for use in the UI. In addition to the *Embedded Workflows* UI, you can now select **Embedded Workflows** as a dropdown option along with **Embedded Automate** for various fields on other UI pages.
+
+#### Embedded Workflow UI
+
+The new *Embedded Workflows* functionality adds the following pages to the UI, which you can use to access and work with workflows. To view these pages, go to **Automation** > **Embedded Workflows** and select the page.
+
+![Embedded Workflow UI](../images/embedworkflow_ui.png){: caption="Embedded Workflow UI" caption-side="bottom"}
+
+- **Workflows**
+
+   This UI page includes a summary of the available workflows. You can use different views to review the workflows:
+
+   - Text view, which is the raw representation (JSON or YAML) of the Amazon State Languages (ASL) code.
+   - Graph view, which shows an SVG visual representation of the workflow structure.
+
+- **Repositories**
+
+   This page lists, and provides a summary of, the added embedded workflow repositories. From this page, you can add, update, and remove repositories.
+   When a repository is added and synchronized, any workflows that are found in the repository can be viewed on the **Workflows** UI page.
+
+- **Credentials**
+
+   Credentials are used by {{ site.data.product.title_short }} for authentication when running embedded workflows against machines, synchronizing with inventory sources, and importing project content from a version control system.
+
+   This page lists, and provides a summary of, the credentials for accessing and using embedded workflows. From this page, you can add, update, and delete credentials.
+
+   These credentials are tied to the provider instance, and are used by the workflow when it runs. The credential types are defined by the provider.
+
+#### UI Field Updates
+{: #field_updates}
+
+Existing integration points are updated for the service catalog creation and management process to support choosing to use **Embedded Workflows** or **Embedded Automate**. This choice can be set when working with service catalog items and service dialogs.
+
+* Service Dialog
+
+   When creating a service dialog, fields that are dynamic now include the option for using **Embedded Workflows** or **Embedded Automate**.
+
+   ![Service Dialog options](../images/embedworkflow_servicedialogdynamic.png){: caption="Service Dialog options" caption-side="bottom"}
+
+* Service Catalog Items
+
+   The provisioning, configuring, or retiring entry points now include the option for using **Embedded Workflows** or **Embedded Automate**.
```

Comment:
```suggestion
   The provisioning, reconfigure, and retirement entry points now include the option for using **Embedded Workflows** or **Embedded Automate**.
```

Reply:
"now include" is similar wording to "new" in that it's going to get stale...wondering if just taking out the "now" is sufficient.  (Similarly for the other sections here).

Reply:
I have made this change.